doc/readme.md

```diff
@@ -0,0 +1,16 @@
+# Trapperkeeper Authorization
```

Comment:
I think having a page like this is really helpful. I noticed us doing it in tk-status and think it helps a newcomer figure walk through the docs.

However, if there's a reason you think an index page is problematic I def want to hear it.